src/Krank/Checkers/IssueTracker.hs

```diff
@@ -22,10 +24,15 @@ githubRE = do
   few (psym ('/' /=))
   "/"
   "issues/"
-  issueNum <- few (psym isDigit)
+  issueNum <- some (psym isDigit)
   optional "/"
+  " " <|> "\t" <|> "\n"
```

Comment:
I think actually the fact that there is a termination "separator" should be more handled by the caller.

I'm not sure though. Because there could be a `https://github.com/foo/bar/issues/666/stats` that should not be matched. The fact that the URL is properly terminated is kind of part of the definition of the issue URL....

I'm conflicted.
WDYT?

Reply:
What about other kind of terminator? If for example the comments use markdown and the issue url is surrounded by parenthesis?

Is matching `https://github.com/foo/bar/issues/666/stats` really an issue?

Could we start with a simple solution: no `optional "/"` neither "termination separator", and see how it behave. Worse case, it will just do some false positive which can be added later in a potential ignore list.